index.md

```diff
@@ -665,97 +665,133 @@ they keep track of what was changed in a file when and by whom,
+copy the entire project into a sub-folder in the area that's being synchronized
+whose name reflects the date.
+This approach results in projects being organized like this:
+
+~~~
+.
+|-- project_name/
+|   \-- current
+|       \-- ...project content as described earlier...
+|   \-- 2016-03-01
+|       \-- ...content of 'current' on Mar 1, 2016
+|   \-- 2016-02-19
+|       \-- ...content of 'current' on Feb 19, 2016
+~~~
 
-While requiring committed discipline from a researcher,
-this manual procedure provides the first three abilities of a version control system
-without the use of any new tool.
+Here,
+the `project_name` folder is mapped to external storage (such as Dropbox),
+`current` is where development is done,
+and other folders within `project_name` are old versions.
+
+Coyping everything like this may seem wasteful,
+since many files won't have changed,
+but consider:
+a terabyte hard drive costs about $50 retail,
+which means that 50 GByte costs less than a latte.
+Provided large data files are kept out of the backed-up area (discussed below),
+this approach costs less than the time it would take to select files by hand for copying.
+
+While requiring discipline from a researcher,
+this manual procedure meets the core requirements outlined above
+without needing any new tools.
 If multiple researchers are working on the same project,
-they will need to coordinate efforts so that only a single person is working at any given time
-and incrementing the version number of the project.
-This procedure does not directly assist with the fifth goal of a good version control system,
-making the project widely available for collaboration.
-
-Once again, we stress that common version control tools will greatly assist a user
-in performing essentially the same tasks as this manual procedure,
-while also providing a formal framework that will better scale to larger projects.
-Abilities to branch, merge, collaborate simultaneously on files, and interact with other software developers
-are not parts of what we consider to be "good enough practices",
-but they are important components of our best practices.
-Novice users are encouraged to graduate from manual systems such as this one
-to one of the major version control tools as soon as is practical.
-
-Regardless of whether a formal tool or a manual approach is chosen,
-the first rule for using version control is that nearly
-everything created by a human being goes under version control as soon as it is created.
+though,
+they will need to coordinate so that only a single person is working on specific files at any time.
+In particular,
+they may wish to create one change log file per contributor,
+and to merge those files whenever a backup copy is made.
+
+Version control tools like Git, Mercurial, and Subversion essentially automate and accelerate
+the manual process just described:
+
+1.  Instead of requiring users to copy everything into subfolders,
```

Comment:
I think the version control parts needs a bit more explanation. It currently assumes the reader knows what it is and how it works. 'Everything is kept in one set of folders without any dates in the names, commit early and commit often, write informative commit messages' are thing that could (should?) be mentioned, before the list of benefits.